dlc-wallet/src/main/scala/org/bitcoins/dlc/wallet/DLCWallet.scala

```diff
@@ -1245,16 +1048,21 @@ abstract class DLCWallet extends Wallet with AnyDLCHDWalletApi {
   private def verifierFromDb(
```

Comment:
not a must but it might be nice to break up all these `_FromDb` methods into a trait like you did with the other stuff.